src/SoundPlayer.hpp

```diff
@@ -60,6 +60,8 @@ struct SoundPlayer
     ZunResult Release(void);
 
     ZunResult LoadSound(i32 idx, char *path);
+    static WAVEFORMATEX *__stdcall GetWavFormatData(i32 *soundData, char *formatString, i32 *formatSize,
```

Comment:
```suggestion
    static WAVEFORMATEX *GetWavFormatData(i32 *soundData, char *formatString, i32 *formatSize,
```

Generally speaking, __stdcall is almost never going to be correct. It's only used by some windows callbacks.